janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -3333,6 +3332,279 @@ public void testRelationTypeIndexes() {
 
     }
 
+    private void createStrictSchemaForVertexProperties() {
+        clopen(option(AUTO_TYPE), "none");
+        VertexLabel label = mgmt.makeVertexLabel("knows").make();
```

Comment:
_knows_ is usually used for edge labels. How about _person_ for vertex labels instead to avoid confusion?